scripts/docker-worker-linux/90-set-up-ephemeral-disks.sh

```diff
@@ -20,9 +20,14 @@ cat << EOF > "${disk_setup_script}"
 # We create an LVM logical volume from all available storage devices,
 # format it, mount it, and ensure basic structure is in place.
 
+# unless we are instructed to use tmpfs
+tmpfs_size=\$(curl --silent http://169.254.169.254/latest/user-data | jq .tmpfsSize | tr -d \")
```

Comment:
maybe we should prefix it with `taskcluster` to be `taskclustertmpfsSize` just to avoid conflicts later or something? No real worries, take it or leave it.